ktor-client/ktor-client-core/common/src/io/ktor/client/response/HttpResponse.kt

```diff
@@ -68,7 +68,7 @@ abstract class HttpResponse : HttpMessage, CoroutineScope, Closeable {
         if (!closed.compareAndSet(false, true)) return
 
         launch {
-            content.discard()
+            content.cancel()
```

Comment:
So we don't wait for content transfer completion but abort it, right? Is it safe?

Reply:
Yes